pkg/reconciler/instances/istio/reset/proxy/proxy.go

```diff
@@ -0,0 +1,58 @@
+package proxy
```

Comment:
To have the same pattern, maybe you could think about having similar action like the performer of the istio install/upgrade